icechunk-python/python/icechunk/repository.py

```diff
@@ -32,9 +30,7 @@ def create(
         return cls(PyRepository.create(storage, config=config))
 
     @classmethod
-    def open(
-        cls, storage: StorageConfig, *, config: RepositoryConfig | None = None
-    ) -> Self:
+    def open(cls, storage: Storage, config: RepositoryConfig | None = None) -> Self:
```

Comment:
Can we go back to having the config be a keyword only argument?

Reply:
Yes, will change. Remind me why you prefer it, I'm indifferent I think

Reply:
It just makes it explicit since it's optional